lib/src/modules/gates.dart

```diff
@@ -679,6 +679,8 @@ class IndexGate extends Module with InlineSystemVerilog {
   /// The bit [index] will be indexed as an output.
```

Comment:
Not from this PR, but mentioning "bit" is not quite correct, maybe call it a signal here since it might be more than 1 bit wide.  Also, it's probably better called a "two-input index gate" without the word "bit" in the line above.